www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -31,6 +32,10 @@ RDEPEND="${COMMON_DEPEND}
 	>=dev-python/pygments-2.1.3[${PYTHON_USEDEP}]
 	>=dev-python/pypeg2-2.15.2[${PYTHON_USEDEP}]
 	dev-python/PyQt5[${PYTHON_USEDEP},declarative,gui,network,printsupport,webkit,widgets]
```

Comment:
Isn't it either webkit or webengine? If it is, then webkit use flag here would need to be made conditional under !webengine, please check the build system